opm/parser/eclipse/EclipseState/Schedule/Well/WellTestState.hpp

```diff
@@ -42,6 +42,10 @@ class WellTestState {
     struct ClosedWell {
         std::string name;
         WellTestConfig::Reason reason;
+        // the well can be re-opened if the well testing is successful. We only test when it is closed.
+        bool closed;
```

Comment:
Since the well can now actually be open - maybe the struct should get a new name - `WTestWell`?

Reply:
Yes. Two people, one mind!